pkg/glusterfs/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Gluster CSI Authors.
```

Comment:
This has the Apache license where as the project has (L)GPL. Is that intended ?

Reply:
same as mentioned in the last comment, license is yet to be sorted out.

Reply:
In that case, it's better to stick to one (GPL for now), than to introduce a mix of both. If need be, this can be changed to BSD license elsewhere.